content/show/coder-radio/0476.md

```diff
@@ -55,11 +55,11 @@
   "podcast_chapters": null,
   "podcast_alt_file": null,
   "podcast_ogg_file": null,
-  "video_file": null,
+  "video_file": "https://201406.jb-dl.cdn.scaleengine.net/coderradio/2022/cr-0476.mp4",
   "video_hd_file": null,
   "video_mobile_file": null,
-  "youtube_link": null,
-  "jb_url": null,
+  "youtube_link": "https://youtu.be/dscw9-81RfA",
+  "jb_url": "/149367/tapping-the-breaks-coder-radio-476/index.html",
```

Comment:
Why do we need this? The old WordPress site is no longer up. Or should this link to https://original.jupiterbroadcasting.net/149367/tapping-the-breaks-coder-radio-476/index.html?

Reply:
It's not used but does provide context to where this information came from for historical purposes.

Reply:
I think leaving them in for historical purposes will be find. Leaving the base URL off makes sense as well because in the off chance that the guys decide they want to host the old site somewhere else, we are not tied to that domain.